### DISCUSSION TITLE
如何修改远程播放地址

### DISCUSSION THREAD

**Discussion by eightsheep67** [monlor/docker-xiaoya#93] [General]
原版xiaoya通过修改docker_address.txt就行了，但是这个版本会自动修改回去，这个不能修改的话，想在非局域网配置tvbox就无法连接xiaoya了，因为地址都是172的

**Comment by monlor** [comment]
tvbox局域网地址本身就是多余的，你用ext在局域网访问就行了

**Reply by eightsheep67** [reply]
> tvbox局域网地址本身就是多余的，你用ext在局域网访问就行了

不改docker address的话，myjson的配置就不对了…非局域网没办法访问

**Reply by monlor** [reply]
用/tvbox/my_ext.json不行么

**Reply by eightsheep67** [reply]
> 用/tvbox/my_ext.json不行么

哦是这个意思，看了应该是可以的

**Reply by kokojacket** [reply]
> 用/tvbox/my_ext.json不行么

docker_address.txt也影响了emby观看夸克资源
![image](https://github.com/user-attachments/assets/adcea89c-08f4-427b-9d64-9e316733580d)

**Reply by monlor** [reply]
> > 用/tvbox/my_ext.json不行么
> 
> 
> 
> docker_address.txt也影响了emby观看夸克资源
> 
> ![image](https://github.com/user-attachments/assets/adcea89c-08f4-427b-9d64-9e316733580d)
> 
> 

你有去实际使用么，不知道一直在这说什么东西？项目现在3w+次安装，都没人有问题 就你有问题？

**Reply by kokojacket** [reply]
> > > 用/tvbox/my_ext.json不行么
> > 
> > 
> > docker_address.txt也影响了emby观看夸克资源
> > ![image](https://private-user-images.githubusercontent.com/74887370/348128794-adcea89c-08f4-427b-9d64-9e316733580d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3NjA5NjUsIm5iZiI6MTcyMDc2MDY2NSwicGF0aCI6Ii83NDg4NzM3MC8zNDgxMjg3OTQtYWRjZWE4OWMtMDhmNC00MjdiLTlkNjQtOWUzMTY3MzM1ODBkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzEyVDA1MDQyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhYjg4NjFjNmI4OGMzOTljNTE4MmQ1ODEwNjNjNDUzNWJiOGZiZjU1N2FlYmE3ODdjYjgyM2Y2YTNkNDdkMzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jh39NZ_IpIk6o3oblVGO9sDPwuEyOsdhDLtHh024JTE)
> 
> 你有去实际使用么，不知道一直在这说什么东西？项目现在3w+次安装，都没人有问题 就你有问题？

我是骂你了还是叼你了？脾气这么大？

**Reply by kokojacket** [reply]
这是讨论区还是无人区？

**Reply by monlor** [reply]
> 这是讨论区还是无人区？

你已经多次重复提问关于docker address的问题，并且你说的问题不存在，不好意思，我已经对你不耐烦了，请你不要出现在我的项目中！没有人有义务要免费且耐心的为你解答问题！

**Reply by kokojacket** [reply]
> > 这是讨论区还是无人区？
> 
> 你已经多次重复提问关于docker address的问题，并且你说的问题不存在，不好意思，我已经对你不耐烦了，请你不要出现在我的项目中！没有人有义务要免费且耐心的为你解答问题！

好的，这其中误会，但我也并不想说了，祝你的项目更好。
